html-forms/index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Html Forms</title>
+  </head>
+  <body>
+    <h1>Coding Form!</h1>
+    <form>
+      <label for="email">Email</label>
+      <br />
+      <input
+        type="email"
+        id="email"
+        name="email"
+        placeholder="codeguy@gmail.com" />
+      <br />
+      <label for="selection">Why is coding cool?</label>
+      <br />
+      <select id="selection" name="selection">
+        <option>Select an option</option>
+        <option>Because it's interesting</option>
+        <option>Because it's fun</option>
+        <option>Because it's powerful</option>
+      </select>
+      <p>Favorite Language (choose one)</p>
+      <input
+        type="radio"
+        id="firstLanguage"
+        value="HTML"
+        name="firstLanguage" />
+      <label for="firstLanguage">HTML</label>
+      <input
+        type="radio"
+        id="secondLanguage"
+        value="CSS"
+        name="secondLanguage" />
+      <label for="secondLanguage">CSS</label>
+      <input
+        type="radio"
+        id="thirdLanguage"
+        value="JavaScript"
+        name="thirdLanguage" />
+      <label for="thirdLanguage"> JavaScript</label>
+      <p>Would you like to sign up for more information?</p>
+      <input type="checkbox" id="moreInfo" name="moreInfo" />
+      <label for="moreInfo">Sign me up!</label>
+      <br />
+      <label>Comments</label>
+      <br />
+      <textarea id="comments" rows="5" cols="25"></textarea>
```

Comment:
This `label` should be linked to the `textarea` on line 53. Missing a `for` attribute on the `label`